.github/workflows/tx-hammer-plugins.yml

```
@@ -0,0 +1,96 @@
+name: Translations
+
+on:
+  workflow_call:
+    inputs:
+      gem_name:
+        description: 'Name of the Gem'
+        required: true
+        type: 'string'
+    secrets:
+      tx_token:
+        description: 'API token for the Transifex.'
+        required: true
+      gh_token:
+        description: 'GitHub token with permissions to create a pull request'
+        required: true
+
+jobs:
+  tx-update:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ${{ inputs.gem_name  }}
+
+    steps:
+      - name: Get hammer-cli
+        uses: actions/checkout@v4
+        with:
+          repository: theforeman/hammer-cli
+          ref: master
+          path: hammer-cli
+
+      - name: Get ${{ github.repository }}
+        uses: actions/checkout@v4
+        with:
+          path: ${{ inputs.gem_name  }}
+
+      - name: Install Transifex dependencies
+        run: |
+          sudo apt-get -y install gettext
+
+      - name: "Install Transifex"
+        run: |
+          curl -o- https://raw.githubusercontent.com/transifex/cli/master/install.sh | bash
+          sudo mv tx /usr/local/bin/
+          sudo chown -R runner:runner /usr/local/bin
+
+      - name: Configure plugin
+        run: |
+          echo "gemspec path: '../hammer-cli', name: 'hammer_cli'" > Gemfile.local.rb
```

I don't see the point of this. Hammer doesn't declare any development dependencies so I'd think this is equivalent:
```suggestion
          echo "gem 'hammer_cli', github: 'theforeman/hammer-cli'" > Gemfile.local.rb
```

Then you don't need to clone it separately.

A further question: how many Gemfiles actually load `Gemfile.local.rb`?